### ISSUE TITLE
Двухфакторная проверка

### ISSUE THREAD

**Issue by proffru** [leontyev-anton/passport.yandex.ru-auth#1]
Антон, добрый день. Пробую использовать ваш код, авторизация не проходит, просит 2ф авторизацию, Вы не обрабатывали этот момент?

**Comment by leontyev-anton** [comment]
**proffru**, спасибо за внимание к моему коду. Нет, не обрабатывал этот момент. В аккаунте с которым работаю она отключена.

**Comment by proffru** [comment]
Да, но сейчас даже при отключённой двухфакторке все равно нужно верифицировать по номеру телефона.

**Comment by proffru** [comment]
Напишите мне в телеграмм @yurij_kozlov

**Comment by leontyev-anton** [comment]
Я пока не очень понимаю, как можно по простому реализовать двухфакторную авторизацию на Python. Ведь код приходит на другое устройство (на телефон).
